client/signup/steps/plans/index.jsx

```diff
@@ -38,6 +38,30 @@ import './style.scss';
 
 export class PlansStep extends Component {
 	componentDidMount() {
+		if ( document && document.createElement && document.body ) {
```

Comment:
Probably wouldn't hurt to confirm `window && window.location` here too.

```
if ( window && window.location && document && document.createElement && document.body )
```

Reply:
A better way of doing this is `typeof document !== 'undefined'` and `typeof window !== 'undefined'`, checking for the existence of `window` and `document` this way will result in referring to undefined variable error, which is complained by the unit test already. Try to run `npm test-client client/signup/steps/plans/test/index.jsx` and you'll know what I mean :)